rtl/include/eclhelper.hpp

```diff
@@ -2779,6 +2780,15 @@ struct IHThorDictionaryResultWriteArg : public IHThorArg
     virtual IHThorHashLookupInfo * queryHashLookupInfo() = 0;
 };
 
+struct IHThorTraceArg : public IHThorFilterArg
```

Comment:
minor: we have tended to avoid inheriting interfaces from one another since it makes it a harder to extend the base interface without invalidating workunits.  In this case I think it is unlikely to cause issues.